web/packages/teleterm/src/ui/Search/pickers/ActionPicker.tsx

```diff
@@ -437,7 +480,61 @@ export function TypeToSearchItem() {
   );
 }
 
+export function ResourceSearchErrorsItem(props: {
+  errors: ResourceSearchError[];
+  getClusterName: (resourceUri: uri.ClusterOrResourceUri) => string;
+  onShowDetails: () => void;
+}) {
+  const { errors, getClusterName } = props;
+
+  let shortDescription: string;
+
+  if (errors.length === 1) {
+    const firstErrorMessage = errors[0].messageWithClusterName(getClusterName);
+    shortDescription = `${firstErrorMessage}.`;
+  } else {
+    const allErrorMessages = errors
+      .map(err => firstLowerCase(err.messageWithClusterName(getClusterName)))
+      .join(', ');
+    shortDescription = `Ran into ${errors.length} errors: ${allErrorMessages}.`;
+  }
+
+  return (
+    <NonInteractiveItem>
+      <Item Icon={icons.Warning} iconColor="#f3af3d">
+        <Text typography="body1">
+          Some of the search results are incomplete.
+        </Text>
+
+        <Flex gap={2} justifyContent="space-between" alignItems="baseline">
+          <span
+            css={`
+              text-overflow: ellipsis;
+              white-space: nowrap;
+              overflow: hidden;
+            `}
+          >
```

Comment:
I'd add `title` here, so you can see what is hidden in ellipsis.

Reply:
I don't think it's warranted in this case – you have the Show Details button which will show what's hidden.